tests/integration/components/cf-field/input-test.js

```diff
@@ -110,4 +110,29 @@ module("Integration | Component | cf-field/input", function(hooks) {
     assert.dom(".uk-form-controls").exists();
     assert.dom("input[type=radio][value='option-1']").isChecked();
   });
+
+  test("it renders a checkbox field", async function(assert) {
+    assert.expect(2);
+
+    await render(hbs`
+      {{cf-field/input
+        field=(hash
+          question=(hash
+            checkboxOptions=(hash
+              edges=(array
+                (hash node=(hash slug="option-1"))
+              )
+            )
+            __typename="CheckboxQuestion"
+          )
+          answer=(hash
+            listValue=(array "option-1")
+          )
+        )
+      }}
```

Comment:
Wouldn't it be more readable to specify those nested objects in JS and pass them in as
`field=field`?

Reply:
That's a matter of taste I think. In this case it would maybe make sense, but I'd prefer to do it everywhere or nowhere in those tests. I think the best way would be to do it everywhere but in a seperate PR. Agree?

Reply:
I created an issue: https://github.com/projectcaluma/ember-caluma-form/issues/38